### ISSUE TITLE
Can't execute View Explorer via key binding after ending a debug session

### ISSUE THREAD

**Issue by 64kramsystem** [microsoft/vscode-cpptools#3012]
**Type: LanguageService**

**Describe the bug**
- OS and Version: Linux Mint 19.1
- VS Code Version: 1.30.1
- C/C++ Extension Version: 0.20.1
- Other extensions you installed (and if the issue persists after disabling them): Go 0.8.0 (Disabled, doesn't affect the issue)
- After running (or debugging) a program, the Explorer view key binding doesn't work.

1. Launch the debug of an empty program (`int main() { return 0 }`). The terminal window and Debug panel will show, the first one being in focus
2. Close the terminal window by pressing the "any" key ;-)
3. Type Ctrl+Shift+E, in order to show again the Explorer panel
4. Instead of the Explorer panel being showed, the `e` character is typed in the editor panel

It's still possible to workaround this problem:

5. Type Ctrl+Shift+D
6. Type Ctrl+Shift+E

The Explorer panel will now show.

**Expected behavior**
The first attempt to show the Explorer windows should succeed, instead of `e` being printed.

**Comment by pieandcakes** [comment]
@saveriomiroddi Does this only happen with the C++ extension? @sean-mcmanus Does the langauge service have a short cut set for `Ctrl+Shift+E` ?

This seems like it should be a [VS Code issue](https://github.com/microsoft/vscode/issues) because our C++ debug adapter doesn't set any specific shortcut keys.

**Comment by 64kramsystem** [comment]
> Does this only happen with the C++ extension?

Hard to say, since I only work in C++. I recently observed the problem also in another case which seemed disconnected from strictly C++ extension functionalities, so likely a VS code problem.

Closing this bug and reopening as VS Code issue.

Thanks for your attention!

**Comment by 64kramsystem** [comment]
Hello, I've reported the bug, and now both maintainers are blaming each other:

https://github.com/Microsoft/vscode/issues/66682

**Comment by 64kramsystem** [comment]
After further research, I found this to be caused by a default 18.04 Ubuntu shortcuts (IBus emoji composer; I don't have IBus, but surely there's something similar).

It's not clear why it's overridden by VS Code sometimes, and why not others, however, I guess the responsibility is on the user to find out how to change the system configuration.

Apologies for the "blaming" term I've used, it was inappropriate.
